### PR TITLE
Limit additional properties to x_* and add supportingMedia to description.

### PR DESCRIPTION
**Pull request by chandanbn**
- Limit additional properties to x_* name.
- Add supportingMedia to description to allow rich-text or supporting images or media.

### CODE REVIEW

File: cve_json_schema/v5.x_discuss/cve513.schema [hunk 1]
```diff
@@ -592,6 +602,35 @@
                   "type": "string",
                   "description": "Description of the vulnerability. Eg., [PROBLEMTYPE] in [COMPONENT] in [VENDOR] [PRODUCT] [VERSION] allows [ATTACKER] to [IMPACT] via [VECTOR]. OR [COMPONENT] in [VENDOR] [PRODUCT] [VERSION] [ROOT CAUSE], which allows [ATTACKER] to [IMPACT] via [VECTOR].",
```
Comment (mprpic) [inline cve_json_schema/v5.x_discuss/cve513.schema:603]: I guess this should now explicitly note that it should be a "**Plain text** description of the vulnerability..."?

Comment (chandanbn) [reply cve_json_schema/v5.x_discuss/cve513.schema]: added it.

**Comment by mprpic** [comment]
@chandanbn Do you know what the status of this PR is? Was this change ever approved by the AWG, or is awaiting some further discussion?

**Comment by chandanbn** [comment]
@mprpic @jwhitmore-mitre This was approved by AWG, and needs to be merged as well.

**Comment by jwhitmore-mitre** [comment]
Applied to new schema repository. Closing down this schema collection.

https://github.com/CVEProject/cve-schema

**Pull request closed without merging**